ranko11103/vjezba3/CMakeList.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.12)
+
+
+project(flipflopproject)
+
+set(SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/d_ff.VHDL
+    ${CMAKE_CURRENT_SOURCE_DIR}/d_ff_testbench.VHDL
+    ${CMAKE_CURRENT_SOURCE_DIR}/t_ff.VHDL
+    ${CMAKE_CURRENT_SOURCE_DIR}/t_ff_testbench.VHDL
+    ${CMAKE_CURRENT_SOURCE_DIR}/rs_latch.VHDL
+    ${CMAKE_CURRENT_SOURCE_DIR}/rs_testbench.VHDL
+    ${CMAKE_CURRENT_SOURCE_DIR}/jk_ff.VHDL
+    ${CMAKE_CURRENT_SOURCE_DIR}/jk_ff_testbench.VHDL
+)
+
+add_library(${PROJECT_NAME} SHARED ${SOURCES})
+
+add_custom_command(
+    TARGET ${PROJECT_NAME} POST_BUILD
+    COMMAND ghdl-gcc ARGS -a ${SOURCES}
+    COMMAND ghdl-gcc -e d_ff_testbench
+    COMMAND ghdl-gcc -e jk_ff_testbench
+    COMMAND ghdl-gcc -e rs_testbench
+    COMMAND ghdl-gcc -e t_ff_testbench
+)
+
+add_test(
```

Comment:
Ime testa treba da se zavrsrsava sa tvojim imenom, npr. D_flipflop_tb_test
Popavi to za svaki test

Reply:
Popravio sam sve prolaze mi i testbench pa pogledajte ako bude sta falilo javite